client/css/_overrides.scss

```diff
@@ -133,7 +133,7 @@ blockquote {
   }
 }
 
-.jobs-list, .companies-list {
+.card-flex {
```

Comment:
Bulma already has helpers for flex, might want to check these out: http://bulma.io/documentation/modifiers/responsive-helpers/

(We're using Bulma.io as the main css "framework" currently)

Reply:
Yeah I'll check those out and replace as much as I can. I had set that .card-flex class because the home page uses the same class names and that was just a way to make sure my css was only touching jobs and companies pages. It was causing issues with the home page.

Reply:
So Bulma is awesome!